gradle.properties

```diff
@@ -8,11 +8,12 @@ archives_base_name = porting_lib
 # is filled in at build
 mod_version = 2.1.<patch>
 
+loom_version = 1.5.+
```

Comment:
What is this for? It's not doing anything since it's not even hooked up in the build.gradles